app/models/target.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id          :bigint           not null, primary key
+#  user_id     :bigint
+#  area_lenght :integer          not null
+#  title       :string           not null
+#  topic       :integer          not null
+#  latitude    :decimal(10, 6)   not null
+#  longitude   :decimal(10, 6)   not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class Target < ActiveRecord::Base
+  enum topic:
```

Comment:
This is good for now, but what would happen if you would like to add a new topic on runtime?
Consider refactoring this to a new model.

Reply:
@gabrielbursztein2 good point! We will do it later when adding the Admin site. Where the admin should be able to add different topics. Thanks anyway